src/sonic-yang-mgmt/tests/libyang-python-tests/sonic_config_data.json

```diff
@@ -0,0 +1,276 @@
+{
+	"test-sonic-vlan:sonic-vlan": {
+		"test-sonic-vlan:VLAN_INTERFACE": {
+			"VLAN_INTERFACE_LIST": [{
+					"vlanid": 111,
+					"ip-prefix": "2000:f500:45:6709::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "10.1.1.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "2000:f500:41:4e9::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "10.1.5.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				}
+			]
+		},
+
+		"test-sonic-vlan:VLAN": {
+			"VLAN_LIST": [{
+					"vlanid": 111,
+					"description": "server_vlan",
+					"dhcp_servers": [
+						"10.1.7.116"
+					],
+					"mtu": "9216",
+					"admin_status": "up",
+					"members": [
```

Comment:
i think this field is deprecated, we are only using vlan_member

Reply:
Again it is test data for sonic_yang_mgmt library, not the actual sonic yang models, I feel, we should provide a single file for the test_yang_models, which should not match with sonic-yang-models, so that there is no confusion regarding it. Let me know if you feel the same. 

Though this point is valid even for https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-vlan.yang, so I will remove it from there, Since it is deprecated.